scripts/get-coreos

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 # USAGE: ./scripts/get-coreos
-# USAGE: ./scripts/get-coreos channel version dest
+# USAGE: ./scripts/get-coreos channel version dest oem_id
 set -eou pipefail
 
 GPG=${GPG:-/usr/bin/gpg}
 
 CHANNEL=${1:-"stable"}
 VERSION=${2:-"1298.7.0"}
 DEST_DIR=${3:-"$PWD/examples/assets"}
+OEM_ID=${4:-""}
```

Comment:
Positional as in this is locked in as the 4th argument. Optional argument would be `-oem_id=""`.

If a 5th argument was added for another optional parameter, you'd have to pass something for the 4th. It starts to get unwieldy.